tee-worker/ts-tests-with-workspaces/.yarnrc.yml

```diff
@@ -0,0 +1,4 @@
+enableGlobalCache: false
+nmHoistingLimits: "workspaces"
+nodeLinker: "node-modules"
```

Comment:
Maybe we could get rid of this if we didn't need to pass the `node_modules/.bin/polkadot-types-from-defs` and `-from-chain` paths to `ts-node` :P